http/responseemitter.go

```diff
@@ -75,7 +77,7 @@ type responseEmitter struct {
 
 	l      sync.Mutex
 	length uint64
-	err    *cmds.Error
+	// err    *cmds.Error
```

Comment:
Field unused, maybe should remove? Or leave commented?

Reply:
Remove. We used to have a `SetError` function but switched to simply returning errors.